build-a-rest-api-frontend/source_code_step_3/static/js/index.js

```diff
@@ -0,0 +1,8 @@
+import { DebugForm } from "./debug.js";
+
+function main() {
+  const debug = new DebugForm();
+  debug.showResponse("");
+}
```

Comment:
@iansedano, I’m a bit lost on where to add the conditional statement for `DebugForm`. Would you add it here, or rather in `debug.js`?

Maybe you could even provide me some sample code?